.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -33,24 +33,58 @@ function add_comment {
         "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/${2}/comments"
 }
 
-BRANCHNAME=$(echo -n "/auto-pr-${pr_number}"| base64)
+
+# The following section is to cancel jobs that may be running for the given PR, before starting to run new tests
+function cancel_queued {
+	  # Cancel tests that are queued up to run in TeamCity
+	  curl \
+		--header "Accept: application/json" \
+		--header "Authorization: Bearer $TEAMCITY_TOKEN" \
+		--header "Content-Type:application/xml" \
+		--data-binary @teamcitycancelparams.xml \
```

Comment:
I think this should be @/teamcitycancelparams

Reply:
oh good catch! was running this locally 🤦‍♀️